server/src/controllers/users/login.ts

```diff
@@ -22,15 +22,12 @@ const login = async (req:Request, res:Response, next:NextFunction) => {
     const verifiedUser = await comparePass(password, hashedPass);
 
     if (!verifiedUser) throw CustomizedError('password Invalid', 403);
-
     else {
       const token = await jwtSign({ id, isAdmin, name });
-      res
-        .status(201)
-        .cookie('token', token)
-        .json({ msg: 'logIn successfully' });
+      res.cookie('token', token);
+      res.send({ msg: 'logIn successfully', user: { isAdmin, id, name } });
```

Comment:
the status code for signup is `201` (created)
```suggestion
      res.status(201).cookie('token', token).json({ msg: 'logIn successfully', user: { isAdmin, id, name } });
```